browser/brave_content_browser_client.cc

```diff
@@ -171,7 +172,7 @@ bool BraveContentBrowserClient::HandleExternalProtocol(
 
   return ChromeContentBrowserClient::HandleExternalProtocol(
       url, web_contents_getter, child_id, navigation_data, is_main_frame,
-      page_transition, has_user_gesture, out_factory);
+      page_transition, has_user_gesture, initiating_origin, out_factory);
```

Comment:
we need to look at what other changes related to this were implemented in the chrome handler and make sure we're taking appropriate action in our handlers

Reply:
It looks like this is just a pass-through, that only results in a string displayed in ExternalProtocolDialog. 

https://chromium.googlesource.com/chromium/src/+/13b66bdf6719bcfedf549ff8ce2b18d2279e0396

I don't have access to https://bugs.chromium.org/p/chromium/issues/detail?id=754304 to see any more details.

Comments added in the code:
<pre>
+  // |initiating_origin| is the origin that initiated the navigation to the
+  // external protocol, and may be null, e.g. in the case of browser-initiated
+  // navigations. The initiating origin is intended to help users make security
+  // decisions about whether to allow an external application to launch.
</pre>